.gitignore

```diff
@@ -12,8 +12,8 @@ IaC/AKS/cluster/charts/smoker/helm-config.yaml
 
 *NGSA_Dashboard.json
 
-**/LodeRunner.Core.xml
-**/LodeRunnerApi.xml
+LodeRunner.Core.xml
+LodeRunnerApi.xml
 
```

Comment:
Should this remain **/[file] so that it would ignore in any path?

Reply:
appears that syntax is unnecessary and is more suited if you want to match any file or directory specifically under a directory.

see first example here: https://git-scm.com/docs/gitignore#_examples

if you wanted to match that file only at root, it would look like this: /LodeRunner.Core.xml